api/models/dataSource.js

```diff
@@ -0,0 +1,12 @@
+const { DataSource } = require("typeorm");
+
+const DbdataSource = new DataSource({
```

Comment:
PascalCase(맨 앞글자를 대문자로 표기하는)는 javascript에서 주로 class, enum 같은 속성에 naming할 때 사용합니다. 즉 DataSource라는 클래스로 만들어진 인스턴스는 camelCase로 표기 할 수 있도록 수정해주시고, 연관되는 코드 전부 반영해주세요~!
```suggestion
const dbDataSource = new DataSource({
```